chapters/functions.tex

```diff
@@ -1670,6 +1670,7 @@ \subsection{Using the Inverse Annotation}\label{using-the-inverse-annotation}
 \section{Function Inlining and Event Generation}\label{function-inlining-and-event-generation}
 
 The annotations listed below affect inlining of functions and the related topic of event generation inside functions.
+For the semantics of the annotations see \cref{notation-for-annotation-definitions}.
```

Comment:
Now that #3621 is introducing sections more specifically on the semantics, I don't think we should call it _semantics_ and link to a section on the notation.
```suggestion
See \cref{notation-for-annotation-definitions} regarding the notation used to describe the annotations.
```